test/performance/odp_timer_accuracy.c

```diff
@@ -1026,7 +1026,8 @@ static int print_stat(test_global_t *test_global)
 			return -1;
```

Comment:
This fix patch could be separated to its own PR, so we get it merged more quickly.

Reply:
Created a new PR for minor fixes: https://github.com/OpenDataPlane/odp/pull/2087